extensions/2.0/Khronos/KHR_physics_rigid_bodies/schema/node.KHR_physics_rigid_bodies.motion.schema.json

```diff
@@ -66,7 +66,7 @@
         },
         "gravityFactor": {
             "type": "number",
-            "description": "A multiplier applied to the global acceleration due to gravity.",
+            "description": "A multiplier applied to the acceleration due to gravity.",
```

Comment:
Reasoning: Gravity need not be global. See [OMI_physics_gravity](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_physics_gravity).